server/routes/login.js

```diff
@@ -1,5 +1,13 @@
 'use strict'
```

Comment:
This is just cut and pasted from the signup handler. Login and signup are two different things. Do you want to create a user everytime someone tries to login?